### PR TITLE
Add relative links in Readme

### PR DESCRIPTION
**Pull request by Nazeeh21**
Fixes issue #3010

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,20 @@ Disclosure](https://img.shields.io/badge/Security-Responsible%20Disclosure-yello
...
+ - [Example](./ReadME.md#example)
+ - [Fastify v1.x and v2.x](./ReadME.md#fastify-v1x-and-v2x)
+ - [Core features](./ReadME.md#core-features)
+ - [Benchmarks](./ReadME.md#benchmarks)
+ - [Documentation](./ReadME.md#documentation)
+ - [Ecosystem](./ReadME.md#ecosystem)
+ - [Support](./ReadME.md#support)
+ - [Team](./ReadME.md#team)
+ - [Hosted by](./ReadME.md#hosted-by)
+ - [License](./ReadME.md#license)
```
Comment (mcollina) [inline README.md:40]: Can you please use the correct casing for `README.md`?

File: README.md [hunk 2]
```diff
@@ -25,6 +25,20 @@ Disclosure](https://img.shields.io/badge/Security-Responsible%20Disclosure-yello
 
 <br />
 
+ - [Requirements](./README.md#requirements)
```
Comment (Eomm) [inline README.md:28]: The issue #3010 refers to this page:
image

Comment (Nazeeh21) [reply README.md:28]: I will add that in `SERVER.md` also.

**Review by Eomm — APPROVED** [review]

**Comment by Nazeeh21** [comment]
Hi @Eomm, is there any other change I need to do in this PR?

File: README.md [hunk 3]
```diff
@@ -25,6 +25,20 @@ Disclosure](https://img.shields.io/badge/Security-Responsible%20Disclosure-yello
...
+ - [Example](./README.md#example)
+ - [Fastify v1.x and v2.x](./README.md#fastify-v1x-and-v2x)
+ - [Core features](./README.md#core-features)
+ - [Benchmarks](./README.md#benchmarks)
+ - [Documentation](./README.md#documentation)
+ - [Ecosystem](./README.md#ecosystem)
+ - [Support](./README.md#support)
+ - [Team](./README.md#team)
+ - [Hosted by](./README.md#hosted-by)
+ - [License](./README.md#license)
```
Comment (Eomm) [inline README.md:40]: Could you move the list under the "hero message" intro?

> An efficient server implies a lower cost of the infrastructure, a better responsiveness under load and happy users.
How can you efficiently handle the resources of your server, knowing that you are serving the highest number of requests as possible, without sacrificing security validations and handy development?

**Review by Eomm — CHANGES_REQUESTED** [review]
We need 2 approvals to get it merged.

I recheck a bit the file and tested the link, the first one is not working `[Requirements](./README.md#requirements)` could you remove it

File: README.md (hunk 3 above)
Comment (Nazeeh21) [reply README.md:40]: Sure, I will make those changes and update the PR.

**Review by Eomm — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**